Assets/Scripts/Game/UserInterfaceWindows/DaggerfallInventoryWindow.cs

```diff
@@ -984,10 +984,10 @@ protected void UpdateAccessoryItemsDisplay()
                 // Update button and panel
                 ImageData image = DaggerfallUnity.Instance.ItemHelper.GetInventoryImage(item);
                 panel.BackgroundTexture = image.texture;
-                if (image.width != 0 && image.height != 0)
-                    panel.Size = new Vector2(image.width, image.height);
-                else
-                    panel.Size = new Vector2(image.texture.width, image.texture.height);

```

Comment:
The code removed and code added look identical here. Could you please reduce PR to code changes only?

Reply:
I tried to (hence the revert and recommit). The changes do not show up in the diff in my ide, even when not ignoring whitespace.
I'm guessing it's an cr/lf issue. My git client's core.autocrlf is set to true. 
I can try again tomorrow.

Reply:
Done.

Reply:
While the file has LF line endings, those few lines have CRLF line endings.
Neither Rider nor VScode would preserve those. So I had to resort to using Notepad :)

Reply:
How odd... I'll see if I can normalise line endings after merge. Thanks for looking at it. :)